main.go

```
@@ -204,10 +240,18 @@ func outputJSON(accounts []AccountInfo) {
 
 func outputTable(accounts []AccountInfo) {
 	table := tablewriter.NewTable(os.Stdout)
-	table.Header("Alias Name", "Account ID")
+	table.Header("ID", "ARN", "Email", "Name", "Status", "Joined Method", "Joined Timestamp")
 
 	for _, account := range accounts {
-		table.Append([]any{account.AliasName, account.AccountID})
+		table.Append([]any{
```

golangci-lint (1.64.8)

Error return value of `table.Append` is not checked

(errcheck)